src/txs/Tx.tsx

```diff
@@ -134,17 +134,13 @@ function Tx<TxValues>(props: Props<TxValues>) {
       if (!key.address || isWalletEmpty) return 0
       if (!(wallet || connectedWallet?.availablePost)) return 0
       if (!simulationTx || !simulationTx.msgs.length) return 0
-      try {
-        const unsignedTx = await lcd.tx.create([{ address: key.address }], {
-          ...simulationTx,
-          feeDenoms: [gasDenom],
-        })
-
-        return unsignedTx.auth_info.fee.gas_limit
-      } catch (error) {
-        console.error(error)
-        return 200_000
```

Comment:
A hardcoded gas fee leads to the "Not enough gas" error after submitting a transaction.